devtools/gradle/settings.gradle

```diff
@@ -1 +1,15 @@
+plugins {
+    id "com.gradle.enterprise" version "3.1"
+}
+
+gradleEnterprise {
```

Comment:
Are there any implication for using this?

Reply:
No, that's the recommended approach for publishing scan builds in scans.gradle.com. see https://docs.gradle.com/enterprise/gradle-plugin/#gradle_6_x_and_later